src/software/pipeline/CMakeLists.txt

```diff
@@ -116,6 +116,20 @@ if(ALICEVISION_BUILD_SFM)
           Boost::filesystem
   )
 
+  #Triangulate
+  alicevision_add_software(aliceVision_triangulate
```

Comment:
Could you rename to "aliceVision_sfmTriangulation" to match the Meshroom node?

Reply:
@mugulmd

Reply:
Sure :)